File: opus-native/src/lib/discover/beauty-categories.ts
export type BeautyCategory =
  | 'barbershop'
  | 'hair_salon'
  | 'nail_salon'
  | 'spa'
  | 'beauty_salon'
  | 'lash_studio'
  | 'brow_bar'
  | 'tattoo_studio'
  | 'massage_therapy'
  | 'wellness_center'
  | 'personal_trainer';

export const BEAUTY_CATEGORIES: { id: BeautyCategory; label: string }[] = [
  { id: 'barbershop', label: 'Barbershop' },
  { id: 'hair_salon', label: 'Hair Salon' },
  { id: 'nail_salon', label: 'Nail Salon' },
  { id: 'spa', label: 'Spa' },
  { id: 'beauty_salon', label: 'Beauty Salon' },
  { id: 'lash_studio', label: 'Lash Studio' },
  { id: 'brow_bar', label: 'Brow Bar' },
  { id: 'tattoo_studio', label: 'Tattoo Studio' },
  { id: 'massage_therapy', label: 'Massage Therapy' },
  { id: 'wellness_center', label: 'Wellness Center' },
  { id: 'personal_trainer', label: 'Personal Trainer' },
];

export function beautyCategoryLabel(id: string | undefined): string | undefined {
  if (!id) return undefined;
  return BEAUTY_CATEGORIES.find((c) => c.id === id)?.label;
}

File: opus-mk-ios/Opus MK/Opus MK/Lib/BeautyCategories.swift
import Foundation

struct BeautyCategoryItem: Identifiable, Hashable {
    let id: String
    let label: String
    let symbol: String
}

enum BeautyCategories {
    static let all: [BeautyCategoryItem] = [
        .init(id: "barbershop", label: "Barbershop", symbol: "scissors"),
        .init(id: "hair_salon", label: "Hair Salon", symbol: "paintbrush"),
        .init(id: "nail_salon", label: "Nail Salon", symbol: "hand.raised"),
        .init(id: "spa", label: "Spa", symbol: "leaf"),
        .init(id: "beauty_salon", label: "Beauty Salon", symbol: "heart"),
        .init(id: "lash_studio", label: "Lash Studio", symbol: "eye"),
        .init(id: "brow_bar", label: "Brow Bar", symbol: "face.smiling"),
        .init(id: "tattoo_studio", label: "Tattoo Studio", symbol: "flame"),
        .init(id: "massage_therapy", label: "Massage Therapy", symbol: "figure.mind.and.body"),
        .init(id: "wellness_center", label: "Wellness Center", symbol: "figure.yoga"),
        .init(id: "personal_trainer", label: "Personal Trainer", symbol: "figure.run"),
    ]

    static func label(for id: String?) -> String? {
        guard let id else { return nil }
        return all.first { $0.id == id }?.label
    }
}
